lib/zclient.h

```diff
@@ -792,6 +804,22 @@ struct zclient_options {
 	bool synchronous;
 };
 
+/* link layer representation for GRE like interfaces
```

Comment:
you've put this new block in the middle of the zclient_options definitions - please move it to after `zclient_options_default` ?

Reply:
ok